src/Tizen.NUI.Components/Style/AlertDialogStyle.cs

```diff
@@ -23,25 +23,26 @@ namespace Tizen.NUI.Components
     /// <summary>
     /// AlertDialogStyle is a class which saves AlertDialog's ux data.
```

Comment:
ux -> UX   as always with abbreviations in description text

Reply:
Thank you for your comment.
It has been updated.